### PR TITLE
Pr 7486 fixes

### PR DESCRIPTION
**Pull request by michaelgugino**

### CODE REVIEW

**Comment by vrutkovs** [comment]
Current status:
1) import triggers don't seem to work - had to hardcode the image names
2) node labels are not set - are these being set by bootstrapper?
3) SDN container fails:
```
Invalid NodeConfig /etc/origin/node/node-config.yaml
  servingInfo.certFile: Invalid value: "/etc/origin/node/server.crt": could not read file: stat /etc/origin/node/server.crt: no such file or directory
  servingInfo.keyFile: Invalid value: "/etc/origin/node/server.key": could not read file: stat /etc/origin/node/server.key: no such file or directory
  servingInfo.clientCA: Invalid value: "/etc/origin/node/ca.crt": could not read file: stat /etc/origin/node/ca.crt: no such file or directory
```

**Comment by vrutkovs** [comment]
Something's up with clonerefs

/retest

**Comment by vrutkovs** [comment]
Remaining issues:
* Install tests are failing, SDN project cannot be created, as API can't be reached after wire aggregator has restarted API. Can't reproduce locally
* Web Console won't come up in CRIO tests - ImagePullBackOff - `openshift/origin-web-console:<commit hash>` was not found

**Comment by vrutkovs** [comment]
bot, retest this please

**Comment by vrutkovs** [comment]
bot, retest this

File: roles/openshift_manage_node/tasks/main.yml [hunk 1]
```diff
@@ -24,7 +24,9 @@
 
 - name: Set nodename for bootstrapped masters
   set_fact:
-    openshift:
+    openshift: {{ openshift | combine(l_openshift_node_update) }}
```
Comment (tomassedovic) [inline roles/openshift_manage_node/tasks/main.yml]: This is missing quotes around the curly braces.

Comment (vrutkovs) [reply roles/openshift_manage_node/tasks/main.yml]: Good catch, thanks. Fixed

**Pull request closed without merging**